### PR TITLE
Select links

### PR DESCRIPTION
**Pull request by SimonLab**
unselect any-na linkedIn links https://github.com/akitten/public-profile-page-url-finder/issues/10
select the right http link when the result of a search is a pdf https://github.com/akitten/public-profile-page-url-finder/issues/9

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab can I merge this?

**Comment by SimonLab** [comment]
yes, all tests passed!

File: lib/finder.js [hunk 1]
```diff
@@ -22,8 +22,13 @@ module.exports = function profile(url, html, next) {
   var i = 0;
   for(i=0; i < titleLinks.length; i++) {
     var link = titleLinks[i].children[0].attribs.href;
```
Comment (nelsonic) [inline lib/finder.js]: if there is no href attribute there will be an error here.

**Comment by nelsonic** [comment]
@SimonLab can you paste the link to the Travis Build? for some reason I cannot see it...

**Comment by nelsonic** [comment]
Which of these: https://travis-ci.org/akitten/public-profile-page-url-finder/pull_requests ?

**Comment by nelsonic** [comment]
@SimonLab latest version is on NPM. :white_check_mark:

**Comment by SimonLab** [comment]
@nelsonic great, thanks! I couldn't find the Travis tests, is it possible that Travis didn't run properly for this PR?

**Pull request merged**